pkg/manifest/ostree_deployment.go

```diff
@@ -513,7 +513,7 @@ func createMountpointService(serviceName string, mountpoints []string) *osbuild.
 	}
 	unit := osbuild.Unit{
 		Description:              "Ensure custom filesystem mountpoints exist",
-		DefaultDependencies:      false,
+		DefaultDependencies:      osbuild.SetServiceDefaultDependencies(false),
```

Comment:
We have a common function for this: `common.ToPtr()`

Reply:
fixed in ddeca965c3dccefb544b2df99be78d68f3637f46